2023-RADIUSS-AWS/JupyterNotebook/aws/eksctl-config.yaml

```diff
@@ -3,33 +3,11 @@ apiVersion: eksctl.io/v1alpha5
 kind: ClusterConfig
 metadata:
   name: jupyterhub
-  region: us-east-2
+  region: us-east-1
```

Comment:
Do we want these changes to remain (or was it testing/should be reverted?) I notice that this is in us-east-1, but the first is us-east-2.

Ideally, we could just use one config, or have identical configs that just vary based on adding the storage class. I think if we come back to this next year (or later) it might be confusing distinguishing the two.

Reply:
After more consideration I don't think it's necessary to have the exact config we use to create the tutorial session in the repo. Plus it will create confusion for the attendees. I'll clean this up and let you know when you can look again.